test/grassmann.jl

```diff
@@ -42,7 +42,7 @@ include("utils.jl")
                 test_reverse_diff = false,
                 retraction_methods = [PolarRetraction(), QRRetraction()],
                 inverse_retraction_methods = [PolarInverseRetraction(), QRInverseRetraction()],
-                basis_types_vecs = basis_types,
+                #basis_types_vecs = basis_types,
```

Comment:
What was the problem with this test?

Reply:
It yields an ambiguous error message when `get_basis(::Grassmann{3,2,ℝ}, ::Array{Float64,2}, ::ProjectedOrthonormalBasis{:gram_schmidt,ℝ})` – but since the other PR does a lot to bases, I would like to fix it over there or maybe it is already fixed thereon.

Reply:
OK, that's reasonable.